lib/rules/value-no-vendor-prefix/index.mjs

```diff
@@ -48,17 +45,21 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		/**
-		 * @type {{ raw: IgnoreValues, unprefixed: IgnoreValues }}
+		 * @typedef {string | RegExp} IgnoreValue
+		 * @type {{ raw: IgnoreValue[], unprefixed: IgnoreValue[] }}
 		 * @todo use Object.groupBy once Node v20 support is dropped
 		 */
 		const groups = { raw: [], unprefixed: [] };
-
-		secondaryOptions?.ignoreValues?.forEach((value) => {
```

Comment:
[suggest] The idiom using `flat()` seems suitable here:

```diff
-	secondaryOptions?.ignoreValues?.forEach((value) => {
+	[secondaryOptions?.ignoreValues ?? []].flat().forEach((value) => {
```

Reply:
Just to be sure, you mean at line 62?

Reply:
No, I think the change above is only necessary and `if...else if` at line 61-62 is unnecessary.

Reply:
I think the following works:

```js
[secondaryOptions?.ignoreValues ?? []].flat().forEach((value) => {
	const useRawValue = isRegExp(value) || (value.startsWith('/') && value.match(/\/i?$/));
	const group = useRawValue ? 'raw' : 'unprefixed';

	groups[group].push(value);
});
```

Reply:
So no type check, I get it.

Reply:
It needs some tweaking, but I see what you mean.